lawnchair/src/app/lawnchair/nexuslauncher/OverlayCallbackImpl.java

```diff
@@ -43,13 +46,11 @@
     private int mFlags;
 
     public OverlayCallbackImpl(Launcher launcher) {
-        SharedPreferences prefs = Utilities.getPrefs(launcher);
+        PreferenceManager prefs = PreferenceManager.getInstance(launcher);
 
         mLauncher = launcher;
         mClient = new LauncherClient(mLauncher, this, new StaticInteger(
-                (prefs.getBoolean(ENABLE_MINUS_ONE,
-                        minusOneAvailable()) ? 1 : 0) | 2 | 4 | 8));
-        prefs.registerOnSharedPreferenceChangeListener(this);
```

Comment:
You can also use `prefs.getMinusOneEnable().addListener()` here to avoid having to route the change event through the launcher object. It works similarly to the shared preferences change listener.